sdk/monitor/opentelemetry-exporter/package.json

```diff
@@ -46,7 +46,15 @@
     "type": "git",
     "url": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/monitor/opentelemetry-exporter"
   },
-"prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "//metadata": {
```

Comment:
I think you might also need to add ` "sdk-type": "client",` somewhere in the top level of package.json as well.

Reply:
Added thanks!